soh/src/code/z_parameter.c

```diff
@@ -1100,6 +1092,18 @@ void func_80083108(GlobalContext* globalCtx) {
                     }
                 }
 
+                for (i = 1; i < ARRAY_COUNT(gSaveContext.equips.buttonItems); i++) {
+                    if ((gSaveContext.equips.buttonItems[i] >= ITEM_SHIELD_DEKU) &&
+                        (gSaveContext.equips.buttonItems[i] <= ITEM_BOOTS_HOVER)) {
```

Comment:
I know you copied this from the original check, but should the shields be included here? I don't think they can get assigned to C-Buttons, but I could be misremembering.

Reply:
I believe they can, but they don't do anything anyway, unlike boots/tunics/swords on C buttons.

Reply:
In the build I've been playing (develop branch from a week or so ago) the shields can _not_ be assigned to buttons, only boots and tunics. However I think it would be nice to have shields assignable to buttons.

Reply:
Line 2949 of `z_player.c` made it look like someone had the intention of adding shields to this at some point, which is presumably why the shields are were originally included here. I don't mind changing it or leaving it as is - I just didn't want to inadvertently mess up anybody's future plans!